plugins/language_clojure.lua

```diff
@@ -0,0 +1,53 @@
+-- Author: https://github.com/RohanVashisht1234/
+-- mod-version:3
+local syntax = require "core.syntax"
+syntax.add {
+  name = "Clojure",
+  comment = ";;",
+  files = {
+    "%.clojure$",
+  },
+  patterns = {
+    { pattern = ';;.*',                                   type = 'comment' },
+    { pattern = ';.*',                                    type = 'comment' },
+    { pattern = { '#"', '"', '\\' },                      type = 'string' },
+    { pattern = { '"', '"', '\\' },                       type = 'string' },
+    { pattern = { '"""', '"""', '\\' },                   type = 'string' },
+    { pattern = 'Retention%s+()[%a_][%w_/]*',             type = { 'keyword', 'literal' } },
+    { pattern = ':[%a_][%w_/%-]*',                        type = { 'keyword2', 'literal' } },
```

Comment:
There is no group here, but `type` is a table.

Reply:
fixed